features/FEATURE_BLE/targets/TARGET_CORDIO/SimpleEventQueue.h

```diff
@@ -73,8 +74,12 @@ struct SimpleEventQueue : EventQueue {
         if (_ble_base == NULL) {
             return false;
         }
-
-        EventNode* next = new (std::nothrow) EventNode(event);
+        void* event_buf = WsfBufAlloc(sizeof(EventNode));
+        if (event_buf == NULL) {
+            error("\r\n%s:%d Cordio WsfBufAlloc out of memory\r\n", __FILE__, __LINE__);
```

Comment:
I think that's a leftover for tracing.

Reply:
it's a unrecoverable error, what do you want to do?